ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -381,4 +382,34 @@ enum CocoaPodUtils {
       throw FileManager.FileError.writeToFileFailed(file: path.path, error: error)
     }
   }
+
+  private static var checkedCocoaPodsVersion = false
+
+  /// At least 1.9.0 is required for `use_frameworks! :linkage => :static`
+  /// - Parameters:
+  ///   - directory: Destination directory for the pods.
+  private static func checkCocoaPodsVersion(directory: URL) {
+    if checkedCocoaPodsVersion {
```

Comment:
It doesn't look exactly correct, because the actual result may depend on `directory` in case if Bundler is used.